docs/spec/spec.md

```diff
@@ -612,7 +612,7 @@ HelloWorldClient securedEP = check new("https://localhost:9090",
 );
 ```
 
-#### 5.1.1.7 Client - Self Signed JWT Auth
+#### 5.1.1.7 Client - self-signed JWT auth
```

Comment:
```suggestion
#### 5.1.1.7 Client - self-signed JWT auth
```
Check the industrial terms.

Reply:
https://google.aip.dev/auth/4111#:~:text=A%20self%2Dsigned%20JWT%20is,considered%20a%20standard%20authentication%20method.